src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py

```diff
@@ -241,42 +323,35 @@ def injectFilesPhEDExCall(self, location, injectData, lfnList):
         actual PhEDEx call for file injection
         """
         xmlData = self.createInjectionSpec(injectData)
+        logging.debug("injectFiles XMLData: %s", xmlData)
 
         try:
             injectRes = self.phedex.injectBlocks(location, xmlData)
-        except HTTPException as ex:
-            # If we get an HTTPException of certain types, raise it as an error
-            if ex.status == 400:
-                # assume it is duplicate injection error. but if that is not the case
-                # needs to be investigated
-                self.blocksToRecover.extend( self.createRecoveryFileFormat(injectData) )
-
-            msg = "PhEDEx injection failed with %s error: %s" % (ex.status, ex.result)
-            raise PhEDExInjectorPassableError(msg)
         except Exception as ex:
-            # If we get an error here, assume that it's temporary (it usually is)
-            # log it, and ignore it in the algorithm() loop
-            msg =  "Encountered error while attempting to inject blocks to PhEDEx.\n"
-            msg += str(ex)
-            logging.error(msg)
-            logging.debug("Traceback: %s", str(traceback.format_exc()))
-            raise PhEDExInjectorPassableError(msg)
-
-        logging.info("Injection result: %s", injectRes)
-
-        if "error" in injectRes:
-            msg = ("Error injecting data %s: %s" %
-                   (injectData, injectRes["error"]))
-            logging.error(msg)
-            self.sendAlert(6, msg = msg)
+            if isinstance(ex, HTTPException):
```

Comment:
Originally it got rid of some code duplication in the except clauses, but when I further cleaned this up now it really doesn't anymore. Will switch back to the double except again.